<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Léonard Dekens | Mathematical models for evolutionary biology</title>

<script src="site_libs/header-attrs-2.25/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html"></a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">Contact</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Léonard Dekens | Mathematical models for evolutionary biology</h1>

</div>


<div id="introduction" class="section level2">
<h2>Introduction</h2>
<p>Since November 2023, I am a Postdoctoral Fellow at the <strong><a href="https://www.crick.ac.uk/">Francis Crick Institute</a> (London)</strong>, in the <strong>[Difrisco lab of Theoretical Biology] (<a href="https://www.crick.ac.uk/research/labs/james-difrisco)__" class="uri">https://www.crick.ac.uk/research/labs/james-difrisco)__</a>. Previously, I was a </strong><a href="https://sciencesmaths-paris.fr/en/">FSMP</a> postdoctoral fellow at the department of Applied Mathematics (MAP5) of the University of Paris__, after defending my PhD of Mathematics in July 2022, under the supervision of <a href="https://vcalvez.perso.math.cnrs.fr/">Vincent Calvez</a> (ICJ, CNRS) and <a href="https://imag.umontpellier.fr/~mirrahimi/">Sepideh Mirrahimi</a> (IMAG, CNRS, Montpellier, France). I hold a B.Sc. in Mathematics from the École Normale Supérieure (ÉNS) de Lyon and a M.Sc. in Mathematics and Applied Mathematics from the Claude Bernard University of Lyon and the ÉNS de Lyon.</p>
<p>From 2022 to 2024, I am a serving member of the Graduate Students Advisory Council (GSAC) of the Society for the <a href="https://www.evolutionsociety.org/">Study of Evolution</a> (SSE). As such, I review applications for the Lewontin Awards (for early PhD students) and Rosemary Grant Awards (advanced PhD students) offered yearly by SSE. I also organize workshops for graduate students during the annual meeting “Evolution”.</p>
</div>
<div id="research-interest" class="section level2">
<h2>Research interest</h2>
<p>My research interest lies at the <strong>interface between evolutionary biology and mathematical models of population dynamics</strong> (mainly deterministic - PDEs). I believe that deterministic mathematical approaches are insightful to explain <strong>complex trait dynamics arising from the interplay of ecological and evolutive phenomena in large populations</strong> and I like to build and analyse models that shed new lights on evolutionary questions.</p>
<p><strong><em>What?</em></strong> During my PhD, I have investigated specifically the <strong>influence of sexual reproduction</strong> (as a contrast to asexual, or clonal, reproduction) on sptial eco-evolutionary dynamics in various settings (fragmented environment, invasive species, changing environments etc…), through <strong>quantitative genetic models</strong>. I am also very interested by the modelling of the <strong>evolution of genetic architecture</strong> and <strong>intricate gene-traits associations</strong>.</p>
<p><strong><em>Why?</em></strong> Although <strong>sexual reproduction is ubiquitous</strong> in multi-cellular organisms (plants, animals, fungi), there is only a handful of techniques that are developed to analyse the eco-evolutionary fate of large sexually reproducing populations characterized by a quantitative trait, like the size of individuals, the stamina of their muscles etc… One of the main analytical difficulties relates to <strong>how sexual reproduction shapes the inheritance process of such complex traits resulting from a large number of genes contributions</strong>, even if each individual gene’s inheritance is well understood. Moreover, as an offspring’s trait can be far from both its parents’ traits, this inheritance process from one generation to the next presents new mathematical challenges, as it involves <strong>non-local models for long-range interactions</strong>.</p>
</div>
<div id="previous-experience-of-a-theoretical-biology-lab" class="section level2">
<h2>Previous experience of a theoretical biology lab</h2>
<p>I am very fortunate and grateful for having spent a year visiting <strong>Prof. <a href="https://www.zoology.ubc.ca/~otto/">Sarah Otto</a>’s lab in the Biodiversity Research Center</strong>, UBC, Vancouver, in immersion before and during my first year of PhD to learn more directly about evolutionary biologists’ interests and questions, in particular about <strong>theoretical relationships between population genetic and quantitative genetic models</strong>. I am thrilled about our vivid overseas collaboration and invite you to consult our recent paper <em>The best of both worlds: combining population genetic and quantitative genetic models</em> (see link below), that proposes a composite model that bridges the two fields.</p>
<p>I aknowledge that, during that time, I have lived and worked in the traditional, unceded, and occupied territory of the xʷməθkʷəy̓əm (Musqueam), Sḵwx̱wú7mesh (Squamish), and səlilwətaɬ (Tsleil-Waututh) peoples.</p>
</div>
<div id="publications" class="section level2">
<h2>Publications</h2>
<ul>
<li><strong>Sharp habitat shifts, evolutionary tipping points and rescue: quantifying the perilous path of a specialist species toward a refugium in a changing environment</strong>, L. Dekens, 2023, <em>(under review)</em>, biorxiv link: <a href="https://www.biorxiv.org/content/10.1101/2023.09.28.559956v1" class="uri">https://www.biorxiv.org/content/10.1101/2023.09.28.559956v1</a>)</li>
<li><strong>The best of both worlds: combining population genetic and quantitative genetic models</strong>, L. Dekens, S.P. Otto and V. Calvez, 2022, <em>Theoretical Population Biology</em>, doi: <a href="https://doi.org/10.1016/j.tpb.2022.10.002" class="uri">https://doi.org/10.1016/j.tpb.2022.10.002</a>, arXiv link: <a href="https://arxiv.org/abs/2111.11142" class="uri">https://arxiv.org/abs/2111.11142</a>)</li>
<li><strong>Dynamics of dirac concentrations in the evolution of quantitative alleles with sexual reproduction</strong>, L. Dekens and S. Mirrahimi, 2022, <em>Nonlinearity</em>, doi: <a href="https://doi.org/10.1088/1361-6544/ac91bb" class="uri">https://doi.org/10.1088/1361-6544/ac91bb</a>, arXiv link: <a href="https://arxiv.org/abs/2111.14814" class="uri">https://arxiv.org/abs/2111.14814</a>)</li>
<li><strong>Evolutionary dynamics of complex traits in sexual populations in a heterogeneous environment: how normal?</strong>, L. Dekens, 2022, <em>Journal of Mathematical Biology</em> (doi: 10.1007/s00285-021-01712-0, read-only published version: <a href="https://rdcu.be/cF7gg" class="uri">https://rdcu.be/cF7gg</a>, arXiv link: <a href="https://arxiv.org/pdf/2012.10115.pdf" class="uri">https://arxiv.org/pdf/2012.10115.pdf</a>)</li>
<li><strong>Front propagation of a sexual population with evolution of dispersion: a formal analysis</strong>, L. Dekens and F.Lavigne, 2021, <em>SIAM Journal of Applied Mathematics</em> (doi: <a href="https://doi.org/10.1137/20M1356129" class="uri">https://doi.org/10.1137/20M1356129</a>, arXiv link: <a href="https://arxiv.org/pdf/2105.02523.pdf" class="uri">https://arxiv.org/pdf/2105.02523.pdf</a>)</li>
<li><strong>Influence of the mode of reproduction on dispersal evolution during species invasion</strong>, V. Calvez, J. Crevat, L. Dekens, B. Fabrèges, F. Kuczma, F. Lavigne and G. Raoul, 2020, <em>ESAIM: ProcS 67 120-134</em> (doi: <a href="https://doi.org/10.1051/proc/202067008" class="uri">https://doi.org/10.1051/proc/202067008</a>, pdf: <a href="https://www.esaim-proc.org/articles/proc/pdf/2020/01/proc206708.pdf" class="uri">https://www.esaim-proc.org/articles/proc/pdf/2020/01/proc206708.pdf</a>)</li>
<li><strong>Multivariate distribution correction of climate model outputs: A generalization of quantile mapping approaches.</strong> L. Dekens, S. Parey, M. Grandjacques and D. Dacunha‐Castelle. , 2017, <em>Environmetrics</em> (doi: <a href="https://doi.org/10.1002/env.2454" class="uri">https://doi.org/10.1002/env.2454</a>)</li>
</ul>
</div>
<div id="talks-in-international-conferences-and-workshops" class="section level2">
<h2>Talks in international conferences and workshops</h2>
<ul>
<li><em>Evolution 2023</em> (Albuquerque, USA, June 2023)</li>
<li><em>RMR 2023 : Modèles mathématiques pour la biologie et la santé</em> (Rouen, June 2023)</li>
<li><em>Mathematical Models of Evolutionary Rescue</em> (Max Planck Institute for Evolutionary Biology, Plön, Germany, June 2023)</li>
<li><em>ANR Integro-Differential Equations from EVolutionary biology</em> (IMAG, Montpellier, October 2022)</li>
<li><em>14th Congress of the European Society of Mathematical Biology 2022</em> (Heidelberg, Germany, September 2022)</li>
<li><em>Congress of the European Society of Evolutionary Biology 2022</em> (Praha, Czech Republic, August 2022)</li>
<li><em>MMEE 2022</em> (Reading, UK, July 2022)</li>
<li><em>Evolution 2022</em> (Cleveland, USA, June 2022)</li>
<li><em>Mathematical models of Ecology and Evolution</em> (Institut Henri Poincaré, Paris, March 2022)</li>
<li><em>Non-Local Models arising from Biology</em> (CIRM, Marseille, October 2021)</li>
<li><em>Virtual Evolution 2021</em></li>
<li><em>Society of Mathematical Biology 2021</em> (Talk Award for Population Dynamics &amp; Evolution)</li>
</ul>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
